quickwit/quickwit-config/src/source_config/mod.rs

```diff
@@ -102,6 +102,16 @@ impl SourceConfig {
             source_params: SourceParams::IngestApi,
         }
     }
+
+    /// Creates the default cli-ingest source config.
+    pub fn cli_ingest_source() -> SourceConfig {
+        SourceConfig {
+            source_id: CLI_INGEST_SOURCE_ID.to_string(),
+            num_pipelines: 1,
+            enabled: true,
+            source_params: SourceParams::void(),
```

Comment:
Might be worth creating a `Cli` source type not to confuse users who run the `source list` command.

Reply:
Yes, we could also make `SourceParams::stdin()` point to a `FileSourceParams{ file_path: Some("/dev/stdin") }` instead of the current None value. 
What do you think?

Reply:
Sounds good.

Reply:
Finally went with your solution. It looked bizarre to see Type `file` for `_ingest-cli-source` in this list for 
```
+--------------------+------------+---------+
|                  Sources                  |
+--------------------+------------+---------+
| ID                 | Type       | Enabled |
+--------------------+------------+---------+
| _ingest-api-source | ingest-api | true    |
+--------------------+------------+---------+
| _ingest-cli-source | ingest-cli | true    |
+--------------------+------------+---------+
```

Reply:
Looks clean.